modules/free/arrow-free-data/src/test/kotlin/arrow/free/FreeTest.kt

```diff
@@ -63,12 +63,18 @@ class FreeTest : UnitSpec() {
 
     val EQ: FreeEq<ForOps, ForId, Int> = Free.eq(IdMonad, idInterpreter)
 
+    val cf: (Int) -> Free<ForId, Int> = { it.free() }
+
+    val EQ1 = Free.eq<ForId, ForId, Int>(Id.monad(), FunctionK.id())
+
+    val G = Gen.int().map(cf) as Gen<Kind<FreePartialOf<ForId>, Int>>
```

Comment:
At some point we need a good solution to recursive generators to actually genereate some sort of ast here :/